src/core/server/core_usage_data/core_usage_data_service.test.ts

```diff
@@ -145,6 +145,9 @@ describe('CoreUsageDataService', () => {
                   "certificateAuthoritiesConfigured": false,
                   "certificateConfigured": false,
                   "cipherSuites": Array [
+                    "TLS_AES_256_GCM_SHA384",
+                    "TLS_CHACHA20_POLY1305_SHA256",
+                    "TLS_AES_128_GCM_SHA256",
```

Comment:
It looks like the default set of cipher suites are changing with this version of Node. Can we update the [`server.ssl.cipherSuites`](https://github.com/elastic/kibana/blob/cc784a0b5a835b37247bd2dddee7051a8fd5742e/docs/setup/settings.asciidoc#L513-L516) setting docs to match?

I expect we'll want something like:

```diff
diff --git a/docs/setup/settings.asciidoc b/docs/setup/settings.asciidoc
index 2f2c87ca9c7..4d97500249c 100644
--- a/docs/setup/settings.asciidoc
+++ b/docs/setup/settings.asciidoc
@@ -512,8 +512,8 @@ In addition to this setting, trusted certificates may be specified via <<server-
 
 | `server.ssl.cipherSuites:`
  | Details on the format, and the valid options, are available via the
-https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT[OpenSSL cipher list format documentation].
-*Default: `ECDHE-RSA-AES128-GCM-SHA256, ECDHE-ECDSA-AES128-GCM-SHA256, ECDHE-RSA-AES256-GCM-SHA384, ECDHE-ECDSA-AES256-GCM-SHA384, DHE-RSA-AES128-GCM-SHA256, ECDHE-RSA-AES128-SHA256, DHE-RSA-AES128-SHA256, ECDHE-RSA-AES256-SHA384, DHE-RSA-AES256-SHA384, ECDHE-RSA-AES256-SHA256, DHE-RSA-AES256-SHA256, HIGH,!aNULL, !eNULL, !EXPORT, !DES, !RC4, !MD5, !PSK, !SRP, !CAMELLIA`*.
+https://www.openssl.org/docs/man1.1.1/man1/ciphers.html#CIPHER-LIST-FORMAT[OpenSSL cipher list format documentation].
+*Default: `TLS_AES_256_GCM_SHA384 TLS_CHACHA20_POLY1305_SHA256 TLS_AES_128_GCM_SHA256 ECDHE-RSA-AES128-GCM-SHA256 ECDHE-ECDSA-AES128-GCM-SHA256 ECDHE-RSA-AES256-GCM-SHA384 ECDHE-ECDSA-AES256-GCM-SHA384 DHE-RSA-AES128-GCM-SHA256 ECDHE-RSA-AES128-SHA256 DHE-RSA-AES128-SHA256 ECDHE-RSA-AES256-SHA384 DHE-RSA-AES256-SHA384 ECDHE-RSA-AES256-SHA256 DHE-RSA-AES256-SHA256 HIGH !aNULL !eNULL !EXPORT !DES !RC4 !MD5 !PSK !SRP !CAMELLIA`*.
 
 | `server.ssl.clientAuthentication:`
  | Controls the behavior in {kib} for requesting a certificate from client

```

Reply:
Fixed 👍

Reply:
I guess we should update the release note to say this as well?

Currently it just says:

> Upgrades the major Node.js version used by and shipped with Kibana from v10 to v12.

How about:

> Upgrades the major Node.js version used by and shipped with Kibana from v10 to v12. This includes support for 3 new cipher suites: TLS_AES_256_GCM_SHA384, TLS_CHACHA20_POLY1305_SHA256, and TLS_AES_128_GCM_SHA256.

Reply:
Yeah that makes sense to me!